src/pyOpenMS/pyopenms/__init__.py

```
@@ -71,42 +81,43 @@
+
+To debug this error, please run ldd (on linux), otool -L (on macOS) or dependency walker (on windows) on
+
+{os.path.join(here, "pyopenms*.so")}
+
+======================================================================
+""")
 
     try:
-        import PyQt5.QtCore
+        import PyQt6.QtCore
     except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)